subsys/bootloader/bl_crypto/Kconfig

```diff
@@ -54,7 +54,7 @@ choice SB_CRYPTO_SIG
 
 	config SB_CRYPTO_CLIENT_ECDSA_SECP256R1
 		bool "Use another image's ECDSA secp256r1 implementation"
-		select BL_SECP256R1_EXT_API_REQUIRED
+		depends on BL_SECP256R1_EXT_API_REQUIRED
```

Comment:
why? A custom configuration our customer might have won't be proper anymore.

Reply:
select gave a dependency loop.

Reply:
understod

Reply:
Changed it now to 

		select BL_SECP256R1_EXT_API_ATLEAST_REQUIRED